examples/mgui_with_threadpoolexec.py

```diff
@@ -13,28 +15,17 @@
 @magic_factory
 def make_widget(
     image: ImageData,
-    min_sigma: Annotated[float, {"min": 0.5, "max": 15, "step": 0.5}] = 4,
-    max_sigma: Annotated[float, {"min": 1, "max": 200, "step": 0.5}] = 120,
+    min_sigma: Annotated[float, {"min": 0.5, "max": 15, "step": 0.5}] = 5,
+    max_sigma: Annotated[float, {"min": 1, "max": 200, "step": 0.5}] = 30,
     num_sigma: Annotated[int, {"min": 1, "max": 20}] = 10,
     threshold: Annotated[float, {"min": 0, "max": 1000, "step": 0.1}] = 0.3,
 ) -> Future[LayerDataTuple]:
```

Comment:
`Future` is not subscriptable, and I believe you want `Awaitable[T]`

```suggestion
) -> Awaitable[LayerDataTuple]:
```